### ISSUE TITLE
Overpass vs. Nominatim - different results

### ISSUE THREAD

**Issue by dlubom** [drolbr/Overpass-API#714]
Hello,

I’ve noticed that when I use this Overpass query (here’s the link: [overpass turbo](https://overpass-turbo.eu/s/1FVH)), which relates to tags for the country of Oman, it seems to return a complete list of tags.

[out:json];
relation(305138);
out tags;

However, when I perform a similar query on Nominatim, I get many more tags.

[Nominatim Demo](https://nominatim.openstreetmap.org/ui/details.html?osmtype=R&osmid=305138&class=boundary)

For instance, it appears to be missing what seems like a key tag, “official_name:en” which is present on Nominatim but not on Overpass.

**Comment by ImreSamu** [comment]
> For instance, it appears to be missing what seems like a key tag, “official_name:en”
> which is present on Nominatim but not on Overpass.

The OSM API indicates that the relation with ID 305138 seems to lack the `official_name:en` key;
* https://www.openstreetmap.org/relation/305138#map=6/21.668/56.030
* https://www.openstreetmap.org/api/0.6/relation/305138

```
<tag k="name:zh-Hans" v="阿曼"/>
<tag k="name:zh-Hant" v="阿曼"/>
<tag k="official_name" v="سلطنة عمان"/>
<tag k="official_name:ar" v="سلطنة عمان"/>
<tag k="official_name:ckb" v="سەڵتەنەتی عومان"/>
<tag k="official_name:cs" v="Sultanát Omán"/>
<tag k="official_name:eo" v="Omana Sultanlando"/>
<tag k="official_name:pl" v="Sułtanat Omanu"/>
<tag k="ref" v="OM"/>
<tag k="state_code" v="OM"/>
<tag k="timezone" v="Asia/Muscat"/>
<tag k="type" v="boundary"/>
<tag k="wikidata" v="Q842"/>
<tag k="wikipedia" v="ar:سلطنة عمان"/>
```

**Comment by lonvia** [comment]
Yes, Nominatim tries to link up administrative boundaries to an appropriate place node and then mixes in the name tags from the place into the response. The linking is somewhat involved. If you want to do it in Overpass, the best strategy is to look for a node with a place tag and the same wikidata tag. No idea if that is possible with OverpassQL. You can also use Nominatim's [lookup endpoint](https://nominatim.org/release-docs/develop/api/Lookup/) to retrieve the extended information given an OSM relation ID. Use `namedetails=1`.

**Comment by mmd-osm** [comment]
>  If you want to do it in Overpass, the best strategy is to look for a node with a place tag and the same wikidata tag. No idea if that is possible with OverpassQL

If you're any the place node is a relation member, the query is pretty fast:

```
(
  relation(305138);
  node(r)[place][wikidata](if:t["wikidata"] == u(t["wikidata"]));
);
out;
```

Checking all place/wikidata nodes is much slower:

```
(
  relation(305138);
  node[place][wikidata](if:t["wikidata"] == u(t["wikidata"]));
);
out;
```
